### DISCUSSION TITLE
allure-cypress : getting error  - 'Converting circular structure to JSON'

### DISCUSSION THREAD

**Discussion by pranit-lqc** [allure-framework/allure2#2650] [Questions & Support] [answered]
We are using Vue 3 component on the UI side.

Version,
"allure-commandline": "^2.29.0",
"allure-cypress": "^3.0.0-beta.7",
"cypress": "13.5.1",

Getting below error:
![image](https://github.com/user-attachments/assets/fe16722c-3199-46a2-abdc-b1cf95bf0556)

without the Allure plugin, we don't see this error.

**Comment by delatrie** [comment]
Is the Vue component passed as an argument of some Cypress command? If so, could you please, post the command here?

**Reply by pranit-lqc** [reply]
No, we are not doing component testing.
I am trying to select the date from 'v-date-picker'.

        cy.get('v-date-picker-header').then(($ele) => {
                const text = $ele.text();
                cy.get('v-date-picker-header > button').click();
                if (date) {
                    let value = new RegExp(`^${text.replace(/\D/g, '')}$`);
                    cy.get('v-date-picker-header > button').contains(value).click();
                }
                cy.get('ul.v-date-picker-years').contains('li', year.toString()).click();
                this._selectMonth(month);
                this._selectDay(date);
            });

**Reply by delatrie** [reply, marked as answer]
Thank you for the reply and the info. This is most probably a bug in allure-cypress.
Here is the issue for you to track: allure-framework/allure-js#1070

**Comment by KanaKBhuvaGit** [comment]
![image](https://github.com/user-attachments/assets/3094db02-ec78-44f8-b1e2-174c7bcf9b66)

It's happening when we loop through elements like calender. Not in the list(Dropdown)

```
cy.get('div.cw-option-area-item').each(($element,index) => {
      let value = $element.text()
      if(value.toLowerCase().trim().includes(String(AreaName).toLowerCase().trim())){
        cy.get('div.cw-option-area-item').eq(index).click()
        return false
      }
    })

```
